src/status_im/accounts/update/publisher.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im.accounts.update.publisher
+  (:require [re-frame.core :as re-frame]
+            [re-frame.db]
+            [status-im.constants :as constants]
+            [status-im.accounts.update.core :as accounts]
+            [status-im.pairing.core :as pairing]
+            [status-im.utils.async :as async-util]
+            [status-im.data-store.accounts :as accounts-store]
+            [status-im.utils.datetime :as datetime]
+            [status-im.transport.message.protocol :as protocol]
+            [status-im.transport.message.contact :as message.contact]
+            [status-im.transport.shh :as shh]
+            [status-im.utils.fx :as fx]))
+
+(defonce polling-executor (atom nil))
+(def sync-interval-ms 120000)
+(def sync-timeout-ms  20000)
+;; Publish updates every 48 hours
+(def publish-updates-interval (* 48 60 60 1000))
+
+(defn publish-update [web3 done-fn]
+  (let [db @re-frame.db/app-db
+        my-public-key (get-in db [:account/account :public-key])
+        peers-count (:peers-count db)
+        now (datetime/timestamp)
+        last-updated (get-in
+                      db
+                      [:account/account :last-updated])]
+    (when (and (pos? peers-count)
+               (not (nil? last-updated))
```

Comment:
can we use here `number?` or `int?`?

Reply:
Thanks, I changed it to `pos?` as I noticed sometimes is `0`, in such cases we don't want to publish a contact update as it means we never updated our info.